src/lib/shared/api/episodes.ts

```diff
@@ -0,0 +1,22 @@
+import type { SBEpisode, Episode } from './types';
+import { transformEpisodeRequest } from './adapters';
+import supabaseClient from './supabase';
+
+export const episodeRead = async (): Promise<Episode[]> => {
```

Comment:
Changed to `episodesList`, so, when someone will import from api directory, it'll be `import { episidesList } from '$lib/shared/api';`. Since people first think about an entity, and then about the method, the names should be easy to find.